roboteam_ai/src/control/controlGoToPos/ControlGoToPosLuTh.cpp

```diff
@@ -111,12 +111,11 @@ ControlGoToPosLuTh::Command ControlGoToPosLuTh::goToPos(RobotPtr robot, Vector2
 
         if (! pidInit) {
             pidInit = true;
-            pid.initialize(1.0/rtt::ai::constants::tickRate);
-            pid.setParams(3.0, 0.0, 0.2, 0.0, 0.0, 0.0);
+            pid.setPID(3.0, 0, 0.5);
         }
 
         Vector2 pidPos = me.posData[toStep];
-        Vector2 vel = pid.posControl(robot->pos, pidPos, robot->vel);
+        Vector2 vel = pid.controlPID2(static_cast<Vector2>(target - robot->pos));
```

Comment:
`controlPID2` Is een vreemde naam

Reply:
vraag maar aan koen. ik heb de oude alleen verwijderd.

Reply:
Can you refactor it maybe to have a better name?